src/Pickles/Pickles.TestFrameworks/NUnit/NUnitExampleSignatureBuilder.cs

```diff
@@ -44,5 +48,8 @@ public Regex Build(ScenarioOutline scenarioOutline, string[] row)
 
             return new Regex(stringBuilder.ToString());
         }
+
+        private static readonly Regex punctuationCharactersRegex = new Regex(@"[\n\.-]+", RegexOptions.Compiled);
```

Comment:
I usually follow the StyleCop rules that instruct us to add fields before methods.